### PR TITLE
Split loadout out of loadout row.

### PR DESCRIPTION
**Pull request by ryan-rushton**
No change in functionality here, I have split out the action buttons into a prop and passed them into the `Loadout` component. This will let us customise the action buttons for the loadouts page, loadout drawer, and the optimiser.

## PR's
- https://github.com/DestinyItemManager/DIM/pull/7552 
- https://github.com/DestinyItemManager/DIM/pull/7555 **(this one)**

### CODE REVIEW

File: src/app/loadout/Loadout.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+import ReactDOM from 'react-dom';
+import { useSelector } from 'react-redux';
+import ItemCategorySection from './loadout-ui/ItemCategorySection';
+import PlugDef from './loadout-ui/PlugDef';
+import Subclass from './loadout-ui/Subclass';
+import styles from './Loadout.m.scss';
+import ModAssignmentDrawer from './mod-assignment-drawer/ModAssignmentDrawer';
+import { createGetModRenderKey } from './mod-utils';
+
+export default function Loadout({
```
Comment (bhollis) [inline src/app/loadout/Loadout.tsx:24]: Would be nice to just rename this (and the file) to something that doesn't clash with Loadout

File: src/app/loadout/Loadout.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+      </div>
+      {loadout.notes && <div className={styles.loadoutNotes}>{loadout.notes}</div>}
+      <div className={styles.contents}>
+        {(items.length > 0 || subclass || savedMods.length > 0) && (
+          <>
+            <div>
+              <Subclass defs={defs} subclass={subclass} power={power} />
+            </div>
+            {['Weapons', 'Armor', 'General'].map((category) => (
+              <ItemCategorySection
```
Comment (bhollis) [inline src/app/loadout/Loadout.tsx:83]: Was this already broken out somewhere?

Comment (ryan-rushton) [reply src/app/loadout/Loadout.tsx:83]: Yeah this PR merges into my other one I had up. I just split the changes across PR's to make the diffs more readable

File: src/app/loadout/Loadout.tsx (hunk 1 above)
Comment (ryan-rushton) [reply src/app/loadout/Loadout.tsx:24]: Yeah will do, I couldnt think of a nice name at the time and its really just displaying a Loadout in its entirety. As I still can't think of something I think I am just going to go with `LoadoutDisplay` unless you have a better idea.

**Pull request merged**